src/make_csv.py

```
@@ -420,7 +441,7 @@ def process_all_feeds(output_root: Path, feed_filter: Optional[str] = None):
             orcahello_dets = get_orcahello_detections(feed)
 
             for det in orcasite_dets:
-
+                pst = format_timestamp_pst(det.timestamp)
```

Ruff (0.14.10)

Local variable `pst` is assigned to but never used

Remove assignment to unused variable `pst`

(F841)